src/views/CountryPreparedness/i18n.json

```diff
@@ -3,12 +3,12 @@
     "strings": {
         "nsPreparednessAndResponseCapacityHeading": "NS Preparedness and Response Capacity",
         "startDateLabel": "Start Date",
-        "perPhaseLabel": "Per Phase",
-        "perCycleLabel": "Per Cycle",
+        "perPhaseLabel": "PER Phase",
+        "perCycleLabel": "PER Cycle",
         "typeOfAssessmentLabel": "Type of Assessment",
         "focalPointNameLabel": "Focal Point",
         "focalPointEmailTitle": "Email",
-        "perAssessmentHeading": "PER Process Results",
+        "perAssessmentHeading": "PER Assessment Results",
```

Comment:
This should be PER Process Results. This was mentioned on https://github.com/orgs/toggle-corp/projects/8/views/23?filterQuery=PER#